setup.py

```diff
@@ -133,7 +175,13 @@ def run_npm_build():
         cmdclass={
             "bdist_egg": bdist_egg if "bdist_egg" in sys.argv else bdist_egg_disabled,
         },
-        package_data={"chemiscope": ["chemiscope/sphinx/static/*"]},
+        package_data={
+            "chemiscope": [
+                "sphinx/static/*",
+                "stcomponent/*",
```

Comment:
does it have to be named `stcomponent`? Or could it be stored in `streamlit/static/` or `streamlit/data/` instead?

Reply:
hey, my 2 cents here - `stcomponent` is actually much more consistent with the structure we have in place for jupyter (that creates `nbextension` in there). Sphinx is different because we create a chemiscope.sphinx package and those are files that live inside the repo, and are not autogenerated from elsewhere.